android/app/src/main/java/org/fossasia/openevent/viewmodels/FeedFragmentViewModel.java

```diff
@@ -0,0 +1,32 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.data.facebook.FeedItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FeedFragmentViewModel extends ViewModel {
```

Comment:
This class is redundant. it is doing nothing of logic except holding the LiveData. Please move all possible logic from Fragment to here

Reply:
I tried moving the API code but it requires `Context` and it does UI changes in the lambda expression. So I will have to pass context and reference of FeedFragment as parameters which is against the ViewModel principles. Any advice as to what I can do here?